container/common/clang.yaml

```diff
@@ -4,7 +4,7 @@ commandTests:
 - name: 'clang-version'
   command: 'clang'
   args: ['--version']
-  expectedOutput: ['clang version 7.0.0.*']
+  expectedOutput: ['clang version 7.0.0 [(]trunk 333878[)]']
```

Comment:
any way we can get this value in some variable that is more visible?

Reply:
Done. Now get value from revision.bzl